www/src/components/guidelines/color/swatch.js

```diff
@@ -0,0 +1,130 @@
+import React from "react"
+import CopyToClipboard from "react-copy-to-clipboard"
+
+import { Box } from "../system"
+import { SrOnly } from "../typography"
+
+export default class Swatch extends React.Component {
+  state = {
+    value: this.props.color.hex,
+    displayCopied: false,
+  }
+
+  toggleCopied = () => {
+    this.setState({
+      displayCopied: true,
+    })
+
+    setTimeout(() => {
+      this.setState({
+        displayCopied: false,
+      })
+    }, 2500)
+  }
+
+  handleClick = event => {
+    event.preventDefault()
+    event.stopPropagation()
+  }
+
+  render() {
+    const { a11yLabel, color, swatchStyle, textColor } = this.props
+
+    return (
+      <Box
+        bg={color.hex}
+        css={{
+          ...swatchStyle,
+          ":hover > .btn-copy": {
+            display: `block`,
+          },
+        }}
+      >
+        {a11yLabel !== `×` && (
+          <Box
+            color={textColor}
+            fontSize={0}
+            position="absolute"
+            fontWeight={0}
+            css={{
+              bottom: 2,
+              left: 3,
+              lineHeight: 1,
+              top: `auto`,
+            }}
+          >
+            {a11yLabel}
+          </Box>
+        )}
+
+        <CopyToClipboard text={this.state.value} onCopy={this.toggleCopied}>
+          <button
+            className="btn-copy"
+            css={{
+              background: `none`,
+              border: 0,
+              bottom: 0,
+              color: `black`,
+              cursor: `pointer`,
+              height: `100%`,
+              left: 0,
+              position: `absolute`,
+              right: 0,
+              top: 0,
+              width: `100%`,
+              zIndex: 1,
+              ":focus .tooltip, :hover .tooltip": {
+                display: `block`,
+              },
+            }}
+            onClick={this.handleClick}
```

Comment:
Can we give these buttons an aria label of the hex color so they aren't empty by default? 

I'm also wishing the widget didn't force keyboard users through every swatch. But we could easily solve that with a skip link past it.

Reply:
> Can we give these buttons an aria label of the hex color so they aren't empty by default?

At least this is fixed in 3017886; also read https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html because I wondered about how to pull of the skip link. I assume we'd want that skip link

- after the each (but the last) color name (== button to open the modal) and before the associated swatches,
- the link title would be "Skip _color name_ swatches and jump to next color",
- every color name button would need an `id` attribute?

Oh, also: Colors are listed by group (primary, secondary, neutral) — would the skip link for the last color in each group have to point to the title of the group?

<img src="https://user-images.githubusercontent.com/21834/59544157-0ce39180-8f10-11e9-83d6-98e7c1bc6c2a.jpg" width=320 />

🤷‍♂ 
Happy to fix this in a follow-up PR!